### ISSUE TITLE
CkStyles doesnt work when 'cssClass' in Settings contains spaces

### ISSUE THREAD

**Issue by SimonPaidla** [techdivision/ckstyles#5]
Actual state: If cssClass in Settings contains more than one class (seperated with a whitespace) it wont work after reloading.

Example configuration:
```
        ....
            'danger':
              label: 'Button Red'
              cssClass: 'btn btn-danger'
```
Target state: Any cssClass that has been set with more than one class should remain set.

Tested with neos-ui version: 3.8.1

**Comment by masilikum** [comment]
This problem is still existing. Since it might be useful in many cases to be able to use multiple classes (e.g. "btn btn-primary"), we should try to find a solution. Until then I will note this issue in the Readme.

**Comment by SimonPaidla** [comment]
Fixed in release v1.1.3.
